server/rosetta/types.go

```diff
@@ -35,7 +36,7 @@ const (
 	StageSyncing = "syncing"
 )
 
-var SupportedOperations = []string{OperationSend, OperationFee, OperationDelegate}
+var SupportedOperations = []string{proto.MessageName(&bank.MsgSend{}), proto.MessageName(&stakingtypes.MsgDelegate{}), OperationFee}
```

Comment:
should we remove this var?

Reply:
We need to give a name to the operation that represents a Fee, it is our special case operation. All the others work by Msg.

Reply:
We could extend the node client interface with SupportedOperations(), and let the implementation provide them. This is so that Allow() will return even the operations that the app-devs add. Then in node client we can get a list of supported operation from the codec. ListImplementations, for cycle them and resolve them via interface registry then try to cast each implementation to rosetta.Msg and there you get the supported operations.